ow_plexil/src/plans/FaultDependenciesModel.xml

```diff
@@ -0,0 +1,95 @@
+<System>
+  <Subsystems>
+
+    <Subsystem Name="Power">
+      <Faults>
+        <Fault Name="LOW_STATE_OF_CHARGE">
+          <Impacts Name="Power" Type="Subsystem"/>
```

Comment:
I'm not sure what it would mean for the power system to be impacted/disabled since there are no operations on it.  I imagine this entry supports `Lookup(IsOperable("Power"))` but I'm not sure where this would be useful...